.github/workflows/publish-corfu.yml

```diff
@@ -0,0 +1,51 @@
+name: CorfDb Artifacts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  publish_corfu_db_artifacts:
+
+    runs-on: ubuntu-latest
+    if: github.event_name == 'push'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+
+      - name: Prepare Corfu
+        run: ./mvnw clean install -Pdocker -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip -T 1C
```

Comment:
Do you think its a good idea to run this command without -DskipTests? I see there are -DskipTests on install and deploy and I would like to understand if the tests are skipped before its deployed for a reason.

Reply:
yes, this step is a preparation step, other actions will do testing and checking.
This task can't be run before all the tests pass (since it's triggered for the master branch), there is no point in running all the checks one more time